app/controllers/issue_templates_controller.rb

```diff
@@ -147,6 +147,56 @@ def set_pulldown
                      should_replaced: setting.should_replaced, default_template: default_template }
   end
 
+  #
+  # List templates associated with tracker and project.
+  # TODO: refactor here. Duplicate with set_pulldown....
+  #
+  def list_templates
+    default_template = nil
+    project_id = @project.id
+    tracker_id = @tracker.id
+    setting = IssueTemplateSetting.find_or_create(project_id)
+    inherit_template = setting.enabled_inherit_templates?
+
+    project_ids = inherit_template ? @project.ancestors.collect(&:id) : [project_id]
+    issue_templates = IssueTemplate.search_by_project(project_id)
+                          .search_by_tracker(tracker_id)
+                          .enabled.order_by_position
+
+    project_default_template = issue_templates.is_default.first
+
+    has_project_default_template = project_default_template.present?
+    default_template = nil
+
+    if has_project_default_template
+      default_template = project_default_template.id
+    end
+
+    if inherit_template
+      inherit_templates = get_inherit_templates(project_ids, tracker_id)
+
+      if inherit_templates.any?
+        inherit_templates.each do |template|
+          next unless template.is_default == true
+          default_template = template unless has_project_default_template
+        end
+      end
+    end
+
+    global_issue_templates = GlobalIssueTemplate.joins(:projects)
+                                 .search_by_tracker(tracker_id)
+                                 .search_by_project(project_id)
+                                 .order_by_position
+
+    render action: '_list_templates',
+           layout: false,
+           locals: { default_template: default_template,
```

Comment:
__[rubocop]__
* Closing hash brace must be on the same line as the last hash element when opening brace is on the same line as the first hash element.